keps/sig-autoscaling/20181106-in-place-update-of-pod-resources.md

```diff
@@ -303,10 +303,14 @@ Pod v1 core API:
 Admission Controllers: LimitRanger, ResourceQuota need to support Pod Updates:
 * for ResourceQuota it should be enough to change podEvaluator.Handler
   implementation to allow Pod updates,
-* Pod's Spec.Containers[i].ResourcesAllocated should be used to be in line with
-  current ResourceQuota behavior which blocks resources before they are used
-  (e.g. for Pending Pods),
-* for LimitRanger TBD.
+* to ensure alignment with current ResourceQuota behavior that blocks resources
+  before they are used (e.g. for Pending Pods), we should do the following:
+  * for requests.[cpu|memory], Pod's max(Spec.Containers[i].Resources.Requests,
+    Spec.Containers[i].ResourcesAllocated) is used to compute Pod aggregate,
+  * for limits.[cpu|memory], Pod's max(Spec.Containers[i].Resources.Limits,
```

Comment:
We would just use `Spec.Containers[i].Resources.Limits` here, rather than max(...  Even though the actual state can lag behind the desired one, we still are able to provide constraints around limits applied to pods.

If we take status into account, it is more complex to reason about, and could result in unintentional errors when making updates if a status update comes in at a similar time.

Reply:
You're right, we don't need this for limits.

Reply:
I've updated the KEP to use desired limits. Resolving the issue.